src/Thinktecture.Relay.Connector.Protocols.SignalR/ConnectorConnection.cs

```diff
@@ -124,7 +135,7 @@ private async Task ConnectAsyncInternal(CancellationToken cancellationToken)
 			try
 			{
 				await _hubConnection.StartAsync(cancellationToken);
-				_connectionId = _hubConnection.ConnectionId;
+				_connectionId = _hubConnection.ConnectionId!;
```

Comment:
We should fallback to the default instead (see declaration of `_connectionId`).

```suggestion
				_connectionId = _hubConnection.ConnectionId ?? string.Empty;
```